src/modules/email.ts

```diff
@@ -0,0 +1,13 @@
+import { RestClient } from 'client';
+import { CMConfig } from 'config';
+import { DeleteEmailRequest, SendEmailRequest } from 'types/codemash.dtos';
+
+export async function sendEmail(request: SendEmailRequest) {
+  const client = new RestClient(CMConfig.getInstance());
+  return client.request(request);
+}
+
+export async function deleteEmail(request: DeleteEmailRequest) {
```

Comment:
https://api.codemash.io/json/metadata?op=DeleteEmailRequest
Deletes an email from queue.

Taip gal reiktu ir vadinti kazkas tokio  deleteEmailFromQueue

Jei requestai skiriasi (na tarkim DeleteEmailRequest yra o reiktu kad jau butu deleteEmailFromQueueRequest), tiesiog request, response reikia pacastinti atitinkamai, kiek teko matyt tai labai paprastai padaroma per typescript.

Reply:
Pridejau i types.d.ts `DeleteEmailFromQueueRequest = DeleteEmailRequest`, galbut reiktu is ten ir exportuot viska

Reply:
Aha liux maciau. Galima is ten exportuoti